spec/models/firmware_registry/rest_api_depot_spec.rb

```diff
@@ -33,9 +33,8 @@
 
     context 'when bad host' do
       it 'managed error is raised' do
```

Comment:
Not related to this PR but `managed error`?  Was that supposed to be `manageiq error`?

Reply:
No idea.